src/http/DescopeClient.swift

```diff
@@ -37,18 +37,22 @@ class DescopeClient: HTTPClient {
         ])
     }
     
-    func otpUpdateEmail(_ email: String, loginId: String, refreshJwt: String) async throws -> MaskedAddress {
+    func otpUpdateEmail(_ email: String, loginId: String, refreshJwt: String, updateOptions: UpdateOptions?) async throws -> MaskedAddress {
         return try await post("otp/update/email", headers: authorization(with: refreshJwt), body: [
             "loginId": loginId,
             "email": email,
+            "addToLoginIDs": updateOptions?.addToLoginIDs,
+            "onMergeUseExisting": updateOptions?.onMergeUseExisting
```

Comment:
Since this behavior is repeating, it would be better to create a new extension function and call it everywhere `updateOptions` were added. See the bottom of the file for examples of other extensions.